### PR TITLE
update German language for ver1.17

### PR DESCRIPTION
**Pull request by crispy-cookie**
updated German language for ver1.17

### CODE REVIEW

File: src/main/resources/assets/modmenu/lang/de_de.json [hunk 1]
```diff
@@ -2,8 +2,101 @@
...
+  "modmenu.toggleFilterOptions": "Filter Optionen wechseln",
+  "modmenu.showingMods.a": "Zeige %s Mods",
+  "modmenu.showingMods.1": "Zeige %s Mod",
+  "modmenu.showingLibraries.a": "Zeige %s Bibliotheken",
+  "modmenu.showingLibraries.1": "Zeige %s Bibliothek",
+  "modmenu.showingModsLibraries.a.a": "Zeige %s Mods and %s Bibliotheken",
+  "modmenu.showingModsLibraries.a.1": "Zeige %s Mods and %s Bibliothek",
+  "modmenu.showingModsLibraries.1.a": "Zeige %s Mod and %s Bibliotheken",
+  "modmenu.showingModsLibraries.1.1": "Zeige %s Mod and %s Bibliothek",
+  "modmenu.minecraft": "Minecraft",
```
Comment (jackassmc) [inline src/main/resources/assets/modmenu/lang/de_de.json]: Please remove all entries that aren't changed.

File: src/main/resources/assets/modmenu/lang/de_de.json [hunk 2]
```diff
@@ -2,8 +2,101 @@
...
+  "option.modmenu.count_children.false": "nicht zählen",
+  "option.modmenu.count_libraries": "Bibliothek: %s",
+  "option.modmenu.count_libraries.true": "Zählen",
+  "option.modmenu.count_libraries.false": "nicht zählen",
+  "option.modmenu.count_hidden_mods": "versteckte Mods: %s",
+  "option.modmenu.count_hidden_mods.true": "Zählen",
+  "option.modmenu.count_hidden_mods.false": "Nicht zählen",
+  "option.modmenu.mod_count_location": "Mod Anzahl: %s",
+  "option.modmenu.mod_count_location.title_screen": "Title Ecke",
+  "option.modmenu.mod_count_location.mods_button": "Mods Schalter",
```
Comment (jackassmc) [inline src/main/resources/assets/modmenu/lang/de_de.json]: Is `Button` commonly localized in German?

File: src/main/resources/assets/modmenu/lang/de_de.json [hunk 3]
```diff
@@ -2,8 +2,101 @@
...
+  "option.modmenu.mod_count_location.title_screen_and_mods_button": "Beides",
+  "option.modmenu.mod_count_location.none": "Keines",
+  "option.modmenu.easter_eggs": "Easter Eggs: %s",
+  "option.modmenu.easter_eggs.true": "Aktiviert",
+  "option.modmenu.easter_eggs.false": "Deaktiviert",
+  "option.modmenu.mods_button_style": "Mods Schalter: %s",
+  "option.modmenu.mods_button_style.classic": "Below Realms",
```
Comment (jackassmc) [inline src/main/resources/assets/modmenu/lang/de_de.json]: Missing translation.

File: src/main/resources/assets/modmenu/lang/de_de.json [hunk 4]
```diff
@@ -2,8 +2,101 @@
...
+  "option.modmenu.mods_button_style.replace_realms": "Realms ersetzen",
+  "option.modmenu.mods_button_style.shrink": "Adjacent",
```
Comment (jackassmc) [inline src/main/resources/assets/modmenu/lang/de_de.json]: Missing translation.

**Review by jackassmc — CHANGES_REQUESTED** [review]

File: src/main/resources/assets/modmenu/lang/de_de.json (hunk 1 above)
Comment (CC044) [reply src/main/resources/assets/modmenu/lang/de_de.json]: I have removed everything that is the same as in en_us.json file.

**Review by jackassmc — CHANGES_REQUESTED** [review]
Please test your changes. To do this you can add the new localization file to the Mod Menu JAR file or compile Mod Menu yourself (there are currently problems with the Terraformers Maven so this is more complicated than usual).

There are two problems:
- Many texts are wider than the buttons
- `: ` is displayed twice (it gets added automatically)

It might be acceptable to just not translate `Button`? Or maybe there is a shorter localized alternative?

**Pull request closed without merging**